extension/content.js

```diff
@@ -282,11 +299,18 @@ $(document).on('click', event => {
 	$target.closest('.js-details-container').toggleClass('refined-github-minimized');
 });
 
+// Handle issue list ajax
+$(document).on('pjax:end', event => {
+	addYoursMenuItem();
```

Comment:
Not a huge deal, but I think it would make sense to move the page checking logic in here (`isPrSearch` and `isIssueSearch`), and just make `addYoursMenuItem` responsible for the DOM manipulation. Would match the pattern we follow in most other places.